server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -263,6 +263,39 @@ function doesClientInformationMatch($clientInformation, $firstName, $lastName, $
 	return $firstNameMatches && $lastNameMatches && $emailAddressMatches && $phoneNumberMatches;
 }
 
+// container/blob naming rules here https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata
+function getContainerName($siteId) {
+	$containerName = null;
+	if($siteId == 1) {
+		$containerName = "nebraska-east-union";
+	} else if ($siteId == 2) {
```

Comment:
nit: I would probably remove that `$containerName` variable, and just have each if statement return their value, i.e.
```php
if ($siteId == 1) {
    return 'nebraska-east-union';
}
```

Reply:
Changed that!